features/machine/machine_healthcheck.feature

```diff
@@ -254,18 +254,9 @@ Feature: MachineHealthCheck Test Scenarios
     Given I obtain test data file "cloud/mhc/mhc_malformed.yaml"
     When I run oc create over "mhc_malformed.yaml" replacing paths:
       | n  | openshift-machine-api |
-    Then the step should succeed
-    And I ensure "mhc-malformed" machine_health_check_machine_openshift_io is deleted after scenario
-
-    Then a pod becomes ready with labels:
-      | api=clusterapi, k8s-app=controller |
-
-    When I run the :logs admin command with:
-      | resource_name | <%= pod.name %>                |
-      | c             | machine-healthcheck-controller |
-    Then the output should contain:
-      | remediation won't be allowed: invalid value for IntOrString  |
-      | total targets: <%= cb.machines.count %>                      | #This covers OCP-29062 - empty selectors watches all machines in cluster
+    Then the step should fail 
+    And the output should contain:
+      | spec.maxUnhealthy in body should match '^((100|[0-9]{1,2})%|[0-9]+)$' | #This covers OCP-29062 - empty selectors watches all machines in cluster
```

Comment:
Will the change works on older OCP versions. e.g, 4.8/4.9 ?

Reply:
yes validated on 4.9 - https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/744984/console

Reply:
Ack